src/popup/components/TransactionDetailsBase.vue

```diff
@@ -53,7 +53,7 @@
         <slot name="tip-url" />
 
         <DetailsItem
-          v-if="contractId"
+          v-if="contractId && contractId.startsWith('ct_')"
```

Comment:
Maybe it will be better to use `Ecnoded.ContractAddress` to compare with?

Reply:
We have the `isContract` helper function located in `src/protocols/aeternity/helpers/index.ts` for this purpose.

Reply:
Fixed